Dockerfile

```diff
@@ -758,12 +757,14 @@ ENV FIREFOX_VERSION="${FF_VER}" \
   LOGFILE_BACKUPS=5 \
   LOGS_DIR="/var/log/cont" \
   VIDEO="false" \
+  AUDIO="false" \
   GRID="true" \
   CHROME="true" \
   FIREFOX="true" \
   MULTINODE="false" \
-  FFMPEG_FRAME_RATE=10 \
-  FFMPEG_CODEC_ARGS="-vcodec libx264 -preset ultrafast -pix_fmt yuv420p" \
+  FFMPEG_FRAME_RATE=30 \
```

Comment:
can you leave `10` here as it was? having 3x frame rate will unexpectedly bigger video files for lots of people

take into account this image is running 5 million tests per month, the impact is huge:
https://datastudio.google.com/u/0/reporting/0B3GcHpfNB6-FRDd3RWo1ZWh4N3M/page/VjTF

Reply:
Changed it back to 10 in the latest commit.